server/.golangci.yml

```diff
@@ -13,7 +10,6 @@ linters-settings:
   golint:
     min-confidence: 0
   govet:
-    check-shadowing: true
```

Comment:
Any reason for changing this?

Reply:
`check-shadowing` is deprecated, now using
```YAML
linters:
  settings:
    govet:
      enable:
        - shadow